packages/db/src/kanel/kanel-kysely-zod-compatibility-hook.cjs

```diff
@@ -9,6 +9,9 @@ const asZodObject = (schemaName) =>
 	`as z.ZodObject<{[K in keyof ${schemaName}]: z.Schema<${schemaName}[K]>}>`;
 
 const replaceSchemaCast = (/** @type {string} */ line) => {
+	// replace undefined with z.string() for tsvector columns
+	line = line.replace(/: undefined\./g, ": z.string().");
+
```

Comment:
by default, `kanel` would make the searchVector column `undefined.optional()`, which is nothing